book/src/paper-wallet/keypair.md

```diff
@@ -0,0 +1,69 @@
+# Keypair Generation and Recovery
+
+Using the `solana-keygen` tool, it is possible to generate new seed phrases for key derivation as well as recover keypairs directly from a seed phrase and optional passphrase. The seed phrase and passphrase together can be used as a paper wallet.
+
+{% hint style="info" %}
+We do not offer advice on how to securely handle your paper wallet. Please research the security concerns carefully.
+{% endhint %}
+
+## Keypair Generation
+
+Keypair creation can be done using the `solana-keygen new` command. The command will walk you through choosing an optional passphrase and will display the generated seed phrase for your paper wallet.
+
+```bash
+solana-keygen new --no-outfile
+```
+
+{% hint style="warning" %}
+If `--no-outfile` is not specified, the default behavior is to write the keypair to `~/.config/solana/id.json`
+{% endhint %}
+
+Full usage details:
+
+```bash
+Generate new keypair file from a passphrase and random seed phrase
+
+USAGE:
+    solana-keygen new [FLAGS] [OPTIONS]
+
+FLAGS:
+    -f, --force            Overwrite the output file if it exists
+    -h, --help             Prints help information
+        --no-outfile       Only print a seed phrase and pubkey. Do not output a keypair file
+        --no-passphrase    Do not prompt for a passphrase
+    -s, --silent           Do not display seed phrase. Useful when piping output to other programs that prompt for user
+                           input, like gpg
+
+OPTIONS:
+    -o, --outfile <PATH>    Path to generated file
+```
+
+## Keypair Recovery
```

Comment:
Documenting `recover` here might be a bit of a foot gun.  The paper wallet use-case should discourage writing the keypair to disk

Reply:
Let's replace it with `pubkey` ?

Reply:
Agreed!